langchain/manifest.json

```diff
@@ -0,0 +1,50 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "7993851f-d36b-40f3-8425-92080f3b9d61",
+  "app_id": "langchain",
+  "display_on_public_website": false,
```

Comment:
Just to be sure: do you want to hide for now?

Reply:
We currently hide this for OpenAI as well, I'm not quite sure the implications of this. cc-ing @yzhan289

Reply:
openai is displayed on our public website https://github.com/DataDog/integrations-core/blob/3c26edc63e3fb240eb6e377220b9bd5382be6a66/openai/manifest.json#L5

Reply:
`display_on_public_website` enables the docs to show up on Datadog Docs (ex. for Snowflake: https://docs.datadoghq.com/integrations/snowflake/)

Reply:
Ahh, thanks for pointing this out, I was looking at an older commit for the OpenAI PR.